app/controllers/bedtime_stories.controller.js

```diff
@@ -94,3 +98,73 @@ exports.createStory = async (req, res) => {
   }
 };
 
+
+// Retrieve all Bedtime Stories from the database.
+exports.getAllStories = (req, res) => {
+  const { language_id, genre_id, country_id, role_id, title, author_id } = req.query;
+
+  let condition = {};
+
+  if (language_id) {
+    condition.language_id = language_id;
+  }
+
+  if (genre_id) {
+    condition.genre_id = genre_id;
+  }
+
+  if (country_id) {
+    condition.country_id = country_id;
+  }
+
+  if (role_id) {
+    condition.role_id = role_id;
+  }
+
+  if(author_id) {
+    condition.author_id = author_id;
+  }
+
+  if (title) {
+    condition.title = { [Op.like]: `%${title}%` };
+  }
+
+  BedtimeStory.findAll({ where: condition })
+    .then((data) => {
+      res.send(data);
+    })
+    .catch((err) => {
+      res.status(500).send({
```

Comment:
should "log" the error to the console before responding, so that at least the problem is recorded in the log output of the backend, e.g., `console.log("When looking up stories with condition " + condition + ": " + err);`